core/corehttp/gateway_handler.go

```diff
@@ -152,12 +152,6 @@ func (i *gatewayHandler) getOrHeadHandler(w http.ResponseWriter, r *http.Request
 		ipnsHostname = true
 	}
 
-	if i.config.BlockList != nil && i.config.BlockList.ShouldBlock(urlPath) {
-		w.WriteHeader(http.StatusForbidden)
-		w.Write([]byte("403 - Forbidden"))
-		return
-	}
```

Comment:
> > this blocking is no longer happening in master. I think this is a bug.
> did you try on :5001? It never happened on :8080

Nevermind, I think you mean that this call to `BlockList.ShouldBlock()` doesn't exist anymore. `BlockList` was completely removed, and we now bind the gateway http handler to the http multiplexer using individual explicit paths (from `corehttp.WebUIPaths`, e.g. `/ipfs/QmU3o9bvfenhTKhxUakbYrLDnZU7HezAVxPM6Ehjw9Xjqy`).

The new code is here: https://github.com/ipfs/go-ipfs/blob/6fdfaaf6e4783ae1be7b348e7a6bc0640982c7df/cmd/ipfs/daemon.go#L404-L415